app/doom3d/.cargo/config.toml

```diff
@@ -2,21 +2,27 @@
 rustflags = ["-C", "link-arg=-Tlink.x", "-C", "llvm-args=-ffast-math"]
 
 [build]
-target = ["riscv32i-unknown-none-elf", "x86_64-unknown-linux-gnu"]
+target = ["riscv32i-unknown-none-elf", "x86_64-unknown-linux-gnu", "x86_64-pc-windows-msvc"]
 
 [alias]
 rbuild = "build --release --target riscv32i-unknown-none-elf"
 xbuild = "build --target x86_64-unknown-linux-gnu"
+wbuild = "build --target x86_64-pc-windows-msvc"
 rb = "rbuild"
 xb = "xbuild"
+wb = "wbuild"
 
 rdoc = "doc --release --target riscv32i-unknown-none-elf"
 xdoc = "doc --target x86_64-unknown-linux-gnu"
+wdoc = "doc --target x86_64-pc-windows-msvc"
 rd = "rdoc"
 xd = "xdoc"
+wd = "wdoc"
 
 xrun = "run --target x86_64-unknown-linux-gnu"
+wrun = "run --target x86_64-pc-windows-msvc"
 xr = "xrun"
+wr = "wrun"
```

Comment:
Thanks for adding this, nice to know it works on Windows as well. I really wish that there was a rust target like "native" that just meant whatever machine you're currently on, in embedded for running tests it seems like that's sometimes useful.